mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,19 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(boolean) {
+  let isHappy = boolean;
+      if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
-}
+    }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
 
-  if (isBigEnough) {
+  if (isBigEnough > 10) {
```

Comment:
in this case your mistake is in misunderstanding of meaning of the variable name `isBigEnough`.
let's split it into separate words: `is big enough` - this looks and sounds like a question, right?
`Is big enough?` and is question is being asked about the number `num`. 
the answer for the question if either `Yes` (true) or `No` (false).
Hence, `isBigEnough` variable should be data type of [Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean).
So, to make it boolean true/false, you should assign it with the result of comparison: 
`let isBigEnough = num > 10`
and then use it as is in `if` statement:

```js
function greaterThan10(num) {
  let isBigEnough = num  > 10;

  if (isBigEnough) {
  .....
  }
  .....
```

Note: this is best common practice to have names for boolean variables start with `is` or `should`
read more here: [Tips on naming boolean variables - Cleaner Code](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)